client/pages/Dashboard/NaturallyRegeneratingForest/NaturallyRegeneratingForest.tsx

```diff
@@ -36,7 +37,11 @@ const NaturallyRegeneratingForest = () => {
       label: i18n.t<string>(`statisticalFactsheets.rowName.${variable}`),
       unit,
 
-      data: columns.map((column) => tableData?.[countryIso][tableName][column][variable].raw),
+      data: columns.map((column) => {
+        const raw = tableData?.[countryIso][tableName][column][variable].raw
```

Comment:
Use TableDatas to get data in all charts if applicable

Reply:
Not applicable in this case

Reply:
Why not ?

Reply:
TableDatas expects `Row`, `Col`, `Table` etc, we have the actual values for those `variable`, `column`, `tableName`,